tests/vxlan/test_vnet_vxlan.py

```diff
@@ -22,16 +23,6 @@
 
 vlan_tagging_mode = ""
 
-def get_vxlan_srcport_range_enabled(duthost):
```

Comment:
how do we run this on older branches?

Reply:
In older versions there was **SAI_VXLAN_SRCPORT_RANGE_ENABLE=1** on **sai.profile** file for following hwsku's

SN3800:
• Mellanox-SN3800-D28C50
• Mellanox-SN3800-C64
• Mellanox-SN3800-D28C49S1 (New 10G SKU)

SN2700:
• Mellanox-SN2700-D48C8

Current approach - we have **DEVICE_METADATA|localhost|vxlan_port_range" = "enable"**  in config_db. 
This config (**vxlan_port_range**) defines whether VXLAN src port range feature is enabled on switch or not.
So we remove old functinality checking there. 

Added other checking instead : **tests/vxlan/__init__.py:52**
VXLAN test will work in 4 modes:
1) **vxlan_port_range** config is "enable", **--udp_src_port** and **--udp_src_port_mask** not passed -> feature enabled, use default port and  mask
2) **vxlan_port_range** config is "disable", **--udp_src_port** and **--udp_src_port_mask** not passed -> feature disabled, do not check src port  at all
3) **vxlan_port_range** config is "disable" but at least **--udp_src_port** or **--udp_src_port_mask** passed ->  feature enabled, use passed port or mask (not passed is default)
1) **vxlan_port_range** config is "enable", **--udp_src_port** or **--udp_src_port_mask** passed -> feature enabled, use passed port and  mask (not passed is default)